### PR TITLE
Portuguese Vmware to OpenStack takeover and engage page

### LINKED ISSUE #7943: [Whitepaper] VMware to OpenStack (Portuguese)] - Build takeover and engage page
## Requirements

Please build a takeover and an engage page based on this [brief](https://docs.google.com/spreadsheets/d/1R1RV2n41mAzAHyQLKV6AQFmFfNFdnzQgwpz5CZY7JTI/edit?usp=sharing) and [the design of the Spanish version](https://github.com/canonical-web-and-design/ubuntu.com/issues/6544):
 
- [ ] build takeover
- [ ] create shared Portuguese form
- [ ] build engage page
- [ ] add takeover to `/takeovers/index.html`
- [ ] add takeover to `/index.html`
- [ ] add engage page to `/engage/index.html`
- [ ] test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- New Portuguese Vmware to OpenStack takeover and engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ and http://0.0.0.0:8001/engage/pt/vmware-para-o-openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the takeover and engage page copy is correct against the [brief](https://docs.google.com/spreadsheets/d/1R1RV2n41mAzAHyQLKV6AQFmFfNFdnzQgwpz5CZY7JTI/edit#gid=2113339190) and [copy sheet](https://docs.google.com/spreadsheets/d/1p0RwL4uZqSh1AA1-bxgz9TyjHLm4eic-SNrzdMCtEl8/edit#gid=1373451093)

## Issue / Card

Fixes #7943

## Screenshots

takeover
image

engage page
image

### CODE REVIEW

**Comment by jpmartinspt** [comment]
Demo: https://ubuntu-com-7978.demos.haus/

File: templates/engage/pt/vmware-para-o-openstack.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+wrapper_template: "engage/_base_engage_markdown.html"
+context:
+     title: "Do VMware ao Charmed OpenStack"
+     meta_description: "Migre do VMware para o Charmed OpenStack para reduzir custos e aumentar a eficiência da infraestrutura."
+     meta_image: https://assets.ubuntu.com/v1/86829a92-Meta+data+img.jpg
+     meta_copydoc: https://docs.google.com/spreadsheets/d/1p0RwL4uZqSh1AA1-bxgz9TyjHLm4eic-SNrzdMCtEl8/edit#gid=1373451093
+     header_title: "Do VMware ao Charmed OpenStack"
+     header_subtitle: "Reduza os custos e aumente a eficiência da sua infraestrutura com adoção de código aberto"
+     header_image: "https://assets.ubuntu.com/v1/93de1fdc-facebook_and_linkedin_banner.jpeg"
```
Comment (steverydz) [inline templates/engage/pt/vmware-para-o-openstack.md:10]:
```suggestion
     header_image: "https://assets.ubuntu.com/v1/2d935f28-openstack-cloud.svg"
```

File: templates/engage/pt/vmware-para-o-openstack.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     header_image_width: "250"
```
Comment (steverydz) [inline templates/engage/pt/vmware-para-o-openstack.md:11]:
```suggestion
     header_image_width: "300"
```

File: templates/engage/pt/vmware-para-o-openstack.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+     header_image_height: "160"
```
Comment (steverydz) [inline templates/engage/pt/vmware-para-o-openstack.md:12]:
```suggestion
     header_image_height: "192"
```

**Comment by pmahnke** [comment]
@steverydz I had just put the social image in the wrong part of the yaml

**Pull request merged**